docs/UsersGuide.asciidoc

```diff
@@ -868,13 +868,46 @@ settings), you have to quote it.
 == Use of the REST API
 
 openQA includes a _client_ script which - depending on the distribution - is
-packaged independently if you just want to interface with an existing openQA
-instance without needing to install the full package. Call `openqa-cli --help`
-for help.
+packaged independently to allow interfacing with an existing openQA instance
+without needing to install openQA itself. Call `openqa-cli --help` for help.
+The sub-commands provide further help, e.g. `openqa-cli api --help` contains
+a lot of examples.
 
-Basics are described in the
-<<GettingStarted.asciidoc#gettingstarted,Getting Started>> guide.
+This section focuses on particular API use-cases. Checkout the
+<<Client.asciidoc#client,openQA client>> section for further information about
+the client itself, how authentication works and how plain `curl` can be used.
 
+=== Finding tests
+
+The following example lists all jobs within the job group with the ID `1`
+and the setting `BUILD=20210707` on `openqa.opensuse.org`:
+
+[source,sh]
+----
+curl -s "https://openqa.opensuse.org/api/v1/jobs?groupid=1&build=20210707" | jq
+----
+
+The tool `jq` is used for pretty-printing in this example but it is also useful
+for additional filtering (see js's
+https://stedolan.github.io/jq/tutorial[tutorial]).
+
+However, openQA's API provides many more filters on its own. These can be used
+by adding additional query parameters, e.g.:
+
+* `ids`/`state`/`result`: Return only jobs with matching ID/state/result.
+  Multiple IDs/states/results can be specified by repeating the parameter or
+  by passing comma-separated values.
+* `distri`/`version`/`build`/`test`/`arch`/`machine`
+  /`worker_class`/`iso`: Return only jobs where the job settings match the
+  specified values like in the example above. Note that it is not possible to
+  filter by arbitrary job settings.
+* `groupid`/`group`: Return only jobs within the job group with the
+  specified ID/name like in the example above.
+* `latest=1`: De-duplicates, so that for the same `DISTRI`, `VERSION`, `BUILD`,
+  `TEST`, `FLAVOR`, `ARCH` and `MACHINE` only the latest job is returned.
+* `limit`/`page`: Limit the number of returned jobs and allow paging.
```

Comment:
The "allow paging" part is not quite clear to me. Like, what values should be specified and what do they mean

Reply:
Would it help to use the work pagination and maybe put a link to https://en.wikipedia.org/wiki/Pagination? It seems quite redundant to explain a term here which should be common to everybody who is using databases.

Reply:
Yes, mention "pagination", not even with a link to wikipedia, that should be obvious. But I think here it would help to explain the difference to the default behaviour and also an example would help.

Reply:
Something like `For example page=2&limit=10 would only show results 11-20`. Or the equivalent that actually happens. That's the part I can't guess despite being familiar with the term